crates/uv/src/commands/tool/upgrade.rs

```diff
@@ -125,22 +124,31 @@ pub(crate) async fn upgrade(
                 debug!("Upgrading `{name}` was a no-op");
             }
             Err(err) => {
-                // If we have a single tool, return the error directly.
-                if names.len() > 1 {
-                    writeln!(
-                        printer.stderr(),
-                        "Failed to upgrade `{}`: {err}",
-                        name.cyan(),
-                    )?;
-                } else {
-                    writeln!(printer.stderr(), "{err}")?;
-                }
-                failed_upgrade = true;
+                errors.push((name, err));
             }
         }
     }
 
-    if failed_upgrade {
+    if !errors.is_empty() {
+        for (name, err) in errors
+            .into_iter()
+            .sorted_unstable_by(|(name_a, _), (name_b, _)| name_a.cmp(name_b))
+        {
+            writeln!(
+                printer.stderr(),
+                "{}: Failed to upgrade {}",
```

Comment:
Previously we only showed the name if the user provided more than one tool. I don't think it's that important, though.

Reply:
Good catch, agree it seems fine to always show the name.